pkg/cmd/server/start/start_master.go

```diff
@@ -218,21 +218,23 @@ func (o MasterOptions) StartMaster() error {
 	// TODO: this should be encapsulated by RunMaster, but StartAllInOne has no
 	// way to communicate whether RunMaster should block.
 	go daemon.SdNotify(false, "READY=1")
```

Comment:
@sttts probably time to nuke this ;-)

Reply:
it's openshift start anyway. That will die soon.

Reply:
And btw, genericapiserver calls the same.